tests/sensors/test_range_finder_sensor.py

```diff
@@ -0,0 +1,145 @@
+import holodeck
+import uuid
+
+from tests.utils.equality import almost_equal
+
+
+sphere_config_count = {
+    "name": "test_range_finder_sensor",
+    "world": "TestWorld",
+    "main_agent": "sphere0",
+    "agents": [
+        {
+            "agent_name": "sphere0",
+            "agent_type": "SphereAgent",
+            "sensors": [
+                {
+                    "sensor_type": "RangeFinderSensor",
+                    "configuration": {
+                        "LazerCount": 4
+                    }
+                }
+            ],
+            "control_scheme": 0,
+            "location": [.95, -1.75, .5]
+        }
+    ]
+}
+
+
+def test_range_finder_sensor_count():
+    """Make sure the range sensor updates after specifying laxer count.
+    """
+    binary_path = holodeck.packagemanager.get_binary_path_for_package("DefaultWorlds")
+
+    with holodeck.environments.HolodeckEnvironment(scenario=sphere_config_count,
+                                                   binary_path=binary_path,
+                                                   show_viewport=False,
+                                                   uuid=str(uuid.uuid4())) as env:
+
+        env.tick(10)
+
+        state = env.tick()
```

Comment:
FYI, (you don't have to change), `state = env.tick(10)` returns state from the last tick